Controls/rtprio-setting.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/bash -ex
```
Suggestion: fail immediately if it doesn't work and print out the commands as it runs, so its easier to debug when it inevitably fails on someone's setup